docs/content/config.md

```diff
@@ -71,6 +71,9 @@ resolver:
 # Optional: wrap nullable input fields with Omittable
 # nullable_input_omittable: true
 
+# Optional: omit generated model fields with `@goField(forceResolver: true)`
+# omit_force_resolver_fields: true
```

Comment:
As someone relatively new to gqlgen, I don't think I would know what this option does without looking at the source code if I only had this documentation as a guide. Is there some way to make the comment here more illustrative, or some way you can provide additional context on when someone may want to do this?

Reply:
Thanks for your reviewing! I fixed at eb6ed62cf3a86229b7289f25a8456571364bd3b9